commerce_coordinator/apps/commercetools/catalog_info/foundational_types.py

```diff
@@ -262,6 +268,60 @@ class TwoUCustomTypes:
         ]
     )
 
+    CART_DISCOUNT_TYPE_DRAFT = TypeDraft(
+        key=TwoUKeys.CART_DISCOUNT_CUSTOM_TYPE,
+        name=ls({'en': "Cart Discount Custom Type"}),
+        resource_type_ids=[ResourceTypeId.CART_DISCOUNT],
+
+        field_definitions=[
+            FieldDefinition(
+                type=CustomFieldStringType(),
+                name=TwoUKeys.CART_DISCOUNT_CLIENT,
+                required=False,
+                label=ls({'en': 'Client'}),
+                input_hint=TypeTextInputHint.SINGLE_LINE
+            ),
+            FieldDefinition(
+                type=CustomFieldEnumType(values=CART_DISCOUNT_CATEGORIES),
```

Comment:
@abdullahwaheed If there is a new category in the future, how will this command ensure that it applies/syncs the new values in CT Enum?

Reply:
currently it won't handle it. It would require changes in our util